source/form.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Седона отзывы</title>
+  </head>
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo">
+        <img src="../source/img/logo-sedona-mobile.svg" width="102" height="84" alt="Логотип сайта">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="main-nav__list nav-list">
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="index.html">
+              Главная
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="photo.html">
+              Фото и видео
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a class="nav-list__link" href="https://htmlacademy.ru/">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-main">
+        <h1 class="visually-hidden">Оставьте свой отзыв</h1>
```

Comment:
not the best heading IMO, would be better smth more explicit, like Feedback form page or alike

Reply:
avoid imperative mood? Renamed.